code/modules/mob/mob.dm

```diff
@@ -1337,6 +1337,9 @@ GLOBAL_LIST_INIT(slot_equipment_priority, list( \
 /mob/proc/IsVocal()
 	return TRUE
 
+/mob/proc/is_oxyloss()
+	return FALSE
```

Comment:
Could you rename this proc to something a bit more descriptive?

Reply:
That should be better